src/test/java/step_definitions/SalesForceMarketing.java

```diff
@@ -286,9 +286,17 @@ public static void changeLeadStatus() {
         }
     }
 
-    @And("Create an order")
-    public void createOrder(){
-        CampaignUtil.createOrder();
+    @And("Create an order and activate in marketing crm")
+    public void create_order_And_Activate() {
```

Comment:
Please follow the same name pattern throughout the file, it should not include the '_'

Reply:
done